modules/ROOT/pages/starting-and-stopping-mule-esb.adoc

```diff
@@ -89,7 +89,9 @@ Include your parameters as script parameters (if running in background, after th
 
 === Passing Parameters to the JVM via the Wrapper Config File
 
-Include your parameters in the configuration file `$MULE_HOME/conf/wrapper.conf`. The distribution includes this file with comments (see fragment below). You can edit this file or add to it as needed.  For more information about the `wrapper.conf` file, consult the Java Service Wrapper  http://wrapper.tanukisoftware.com/doc/english/properties.html[online documentation]  for the file.
+Include your parameters in the configuration file `$MULE_HOME/conf/wrapper.conf`. The distribution includes this file with comments (see fragment below). You can edit this file or add to it as needed.
```

Comment:
Suggestions - remove (see fragment below) and put: "...includes this file with comments, as shown in the fragment below." If you want to keep the text in the parentheses, add "the": "see the..."